packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -66,12 +66,21 @@ const ENVS_TO_EXPOSE = [
     'aws_lambda_log_stream_name',
     'aws_region',
     'bundle_id',
+    // These "customer" defined environment variables are set and expected 
+    // by the MRT smoke test suite
+    'customer_defined_array',
+    'customer_defined_boolean',
+    'customer_defined_float',
+    'customer_defined_integer',
+    'customer_defined_json',
+    'customer_defined_null',
+    'customer_defined_string',
     'deploy_id',
     'deploy_target',
     'external_domain_name',
     'mobify_property_id',
     'node_env',
-    'tz'
+    'tz',
```

Comment:
I think this fails the `lint` check in CI, fixable with `npm run lint:fix` after running `npm ci` from `pwa-kit` root